src/components/VideoPostCard.tsx

```diff
@@ -475,47 +476,51 @@ export function CompactVideoPostCard({
             />
             <MediaInsetBorder />
 
-            <View style={[a.absolute, a.inset_0]}>
+            <View style={[a.absolute, a.inset_0, a.shadow_sm]}>
```

Comment:
The base shadow atoms don't have colors yet, anything color related comes from the `Theme` object from `useTheme` e.g. `const {t} = useTheme()` and `t.atoms.shadow_sm`. So gotta use the shadow atom from the theme itself.

Our system is a little weird, apologies 😅 you can think of `t.atoms` as being "color themed atoms". The base `a` atoms are agnostic of theme (and therefore colors) entirely.

Reply:
Yeah we investigated a few options and landed on this in early Jan 2024. It roughly follows Tailwind naming conventions.

An yeah, `t.atoms` always match the active color theme.

Reply:
Cool thanks for the context, I've updated to use the theme atoms. Is the new change what you expected?